udp-reassembly-offload.md

```diff
@@ -27,6 +27,25 @@ This section describes necessary updates in the Windows network stack to support
 
 The NDIS interface for URO is used for communication between TCPIP and the NDIS miniport driver.
 
+## Rules
+
+URO can only be attempted on a batch of packets that meet all the following criteria:
+
+- Unicast only (no broadcast or multicast)
```

Comment:
Do we see a practical reason why NICs _must not_ coalesce broadcast/multicast? This seems like something that is easy to uncoalesce in SW if/when necessary.

Reply:
We don't want broadcast/multicast packets coalesced with other UDP flows, right?  From a hardware standpoint, is it harder to coalesce certain packet types instead of coalescing all types?

Reply:
To be clear, we're not mandating that NICs coalesce broadcast/multicast, we're simply not prohibiting it. I don't have any arguments for why coalescing a broadcast/multicast flow is inherently more harmful than unicast.